libs/src/sdk/api/tracks/constants.ts

```diff
@@ -0,0 +1,16 @@
+export const TRACK_REQUIRED_PROPERTIES = [
+  'cover_art_sizes',
+  'credits_splits',
+  'file_type',
+  'genre',
+  'is_premium',
+  'length',
+  'mood',
+  'owner_id',
+  'premium_conditions',
+  'release_date',
+  'tags',
+  'title'
+]
+
+export const TRACK_REQUIRED_VALUES = ['owner_id', 'title']
```

Comment:
I think mood and genre might ought to be required as well? (at least based on our UI)

Reply:
Yes will add these!